### PR TITLE
Add interactive-ui

### PR DESCRIPTION
**Pull request by awaisanwar544**
### Demo Video

https://user-images.githubusercontent.com/87834222/168282301-b65266dc-aba9-4aaa-a243-35dd9ab9cca8.mov

### Implementation
`app.rb` has methods that do the following:
- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
+require './book'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  attr_reader :persons, :books, :rentals
+
+  def initialize
+    @persons = []
```
Comment (ahmetbozaci) [inline app.rb]: OPTIONAL
- [x] You can fix the typo in `persons`. It should be `people`

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    @books = []
+    @rentals = []
+  end
+
+  def choose_option(operation)
+    case operation
+    when '1'
+      list_all_books
+    when '2'
+      list_all_persons
```
Comment (ahmetbozaci) [inline app.rb]: OPTIONAL
- [x] You can fix the typo in `persons`. It should be `people`

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @awaisanwar544  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] To make a professional readme please fix the broken link in the license section.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- You have one commit for everything. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
___

### Cheers and Happy coding! 💻 👏

almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**